st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
Yeah, but the operation is idempotent, so multiple calls are not a problem, right?

And this way, we can be sure the exchanges will always be present - no surprises if user doesn't use launchdev / st2_deploy, etc.

And we can still leave the script, in case, for some reason user wants to also run it manually.